src/__test__/unit/Utils.spec.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-undef */
```

Comment:
Why does every test file have this prefix?

Reply:
Because we are not importing `describe` and `it` functions.
And `eslint` returns an error.

Reply:
Why not import it?

Reply:
It works under the hood by running `*.spec.js` files in folder `__test__`.
We are using `create-react-app` https://github.com/facebook/create-react-app/issues/2564

Reply:
Alright then if you say so. What about disabling that rule for the whole `__test__` folder instead of prefixing every file on its own? Something like this could work for us: https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy